.github/workflows/ci.yml

```diff
@@ -39,67 +39,6 @@ jobs:
     steps:
       - run: echo "This CI step only runs on the main repo, where we have access to credentials."
 
-  docs-changes:
```

Comment:
I moved this so it is directly before `doc-snippets` since that's now the only job that relies on this.